docs/pages/deploy-a-cluster/high-availability.mdx

```diff
@@ -61,7 +60,7 @@ flowchart
    Developers & server1 & server2 & server3 --> lb1[Load Balancer]
    lb1 --> proxies
 
-   subgraph teleport[Private Network]
+   subgraph teleport[Private Network &nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp]
```

Comment:
This pushes the label so it's not obstructed by the lines.

Reply:
Is there a more elegant way to achieve this? If not, I think we should add a comment above this line indicating why we have all of the non-breaking spaces.

Reply:
There is, but only for left-right diagrams. See https://github.com/mermaid-js/mermaid/issues/1209 for discussion on this issue. I'll add the comment as suggested.